src/OrleansRuntime/Catalog/Catalog.cs

```diff
@@ -171,7 +174,8 @@ public override void GetObjectData(SerializationInfo info, StreamingContext cont
             MultiClusterRegistrationStrategyManager multiClusterRegistrationStrategyManager,
             IStreamProviderRuntime providerRuntime,
             IStreamProviderManager providerManager,
-            IServiceProvider serviceProvider)
+            IServiceProvider serviceProvider,
+			CachedVersionSelectorManager versionSelectorManager)
```

Comment:
nit nit: some spacing weirdness here - looks like the second line is using tabs instead of spaces.